io.go

```diff
@@ -1459,7 +1460,7 @@ func ReadGbkMulti(path string) []Sequence {
 	return sequences
 }
 
-// ReadGbkFlat reads flat genbank files, like the ones provided by the NCBI FTP server
+// ReadGbkFlat reads flat genbank files, like the ones provided by the NCBI FTP server (after decompression)
 func ReadGbkFlat(path string) []Sequence {
 	file, err := ioutil.ReadFile(path)
 	if err != nil {
```

Comment:
Is this only returning if err != nil? return statements aren't supposed to be in control flow stuff. Normally you'd just use control flow to determine what to set the return variable as and then return it at the end of the function. This can mess with some linters otherwise.

Reply:
Fixed.